rskj-core/src/test/java/co/rsk/peg/BridgeSupportProcessFundsMigrationTest.java

```diff
@@ -167,7 +168,7 @@ void test_processFundsMigration(
             Coin.COIN,
             oldFederation.getAddress()
         ));
-        when(provider.getOldFederationBtcUTXOs()).thenReturn(sufficientUTXOsForMigration);
+        when(federationStorageProvider.getOldFederationBtcUTXOs()).thenReturn(sufficientUTXOsForMigration);
```

Comment:
probably i'd have to add `feePerKbSupport` with that `federationStorageProvider` to the built `bridgeSupport`. So i'll do it when that feature is added.
cc: @jeremy-then